scripts/build_summary/failure.py

```diff
@@ -373,6 +391,24 @@ def scan(self):
         return False
 
 
+class ArtifactArchiveFailure(Failure):
+    description = "Failure related to storing data generated by a build"
+    category = "C8 RE Infra"
+
+    def scan(self):
+        match_strings = [
+            "Warning: failed to create container",
+            ("This server could not verify that you are authorized to "
+             "access the document you requested"),
+        ]
+        for line in self.build.log_lines:
+            for str in match_strings:
+                if str in line:
+                    self.matches = True
+                    self.detail = str.strip()
+                    break
+
```

Comment:
It's not breaking out of the outer loop if there is a match, is that intentional?

Reply:
Replaced double loop with product.